src/App.js

```diff
@@ -129,6 +171,28 @@ class App extends Component {
           <div className="Title">
             { calculatorInfo && calculatorInfo.title}
           </div>
+          <div className="Content Stack">
+            <div className="Control">
+              <input 
+                type="text" 
+                value={this.state.newExclusionReason}
+                onChange={this.handleReasonChange}
+                placeholder="Enter a date exclusion reason">
+              </input>
```

Comment:
non-blocking: 
this tag can be self-closing, `/>` which would mean you can git rid of the `>` on 180